pkg/terraform/executor.go

```diff
@@ -106,7 +106,6 @@ func NewExecutor(conf Config) (*Executor, error) {
 	ex.logger = log.WithFields(log.Fields{
 		"phase": "infrastructure",
 	})
-
```

Comment:
This has nothing to do with the commit, please put in a separate commit if you think it improves readability.